test/test.js

```diff
@@ -50,6 +51,20 @@ test('generate screenshots', async t => {
 	t.true((await getStream.buffer(streams[0])).length > 1000);
 });
 
+test('success message', async t => {
+	const stub = sinon.stub(console, 'log');
+
+	const pageres = new Pageres().src('yeoman.io', ['480x320', '1024x768', 'iphone 5s']);
+
+	await pageres.run();
+
+	pageres.successMessage();
+
+	t.is(stub.firstCall.args[0], '\n✔ Generated 3 screenshots from 1 url and 1 size');
+
+	stub.restore();
+});
```

Comment:
It's hard to have a strict standard about this stuff because it's usually so ambiguous. I think the above looks good though.